app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"
+            }
```

Comment:
please, use a code style as mentioned in this file:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md

The curly braces must be located in one of two options: open and start with the text or have a line break between the text